pegasus/sites.v3/hourofcode.com/public/how-to/index.md.partial

```diff
@@ -100,14 +100,12 @@ Provide headphones for your class, or ask students to bring their own, if the tu
 
 ## Other Hour of Code resources for educators:
 - Visit the [Hour of Code Teacher Forum](http://forum.code.org/c/plc/hour-of-code) to get advice, insight and support from other educators.
-<% if @country == 'us' %>
```

Comment:
Are we okay losing this condition?

Reply:
Oh, I meant to ask you about that!

It seems to me like this condition is unnecessary, but I'm not sure who the stakeholders are for this page that can make that call. Do you?